src/app/shared-services/location/location.service.ts

```diff
@@ -17,9 +17,17 @@ import { TranslateService } from '@ngx-translate/core'
 import { LogfileService } from '../logfile/logfile.service'
 import { LogEventScope, LogEventType } from '../logfile/types'
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Yes, otherwise there is the following error:
```
Variable name `App` must match one of the following formats: camelCase, UPPER_CASEeslint[@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention)
```